integrations/extensions/docs/elasticsearch-install-and-setup/ICD_Elasticsearch_install_and_setup.md

```diff
@@ -0,0 +1,257 @@
+  log   [01:19:33.132] [info][status][plugin:elasticsearch@<kibana_version>] Status changed from yellow to green - Ready
+  log   [01:19:33.378] [info][listening] Server running at http://0.0.0.0:5601
+  ```
+
+## Step 3: Create an Elasticsearch index (keyword-search)
+This step is to create an Elasticsearch index with default settings for quick testing and verification.
+With default settings, an Elasticsearch index does keyword search.
+
+* Open http://0.0.0.0:5601 in browser and log into Kibana using the `username` and `password` from the service credentials of the Elasticsearch instance
+* Navigate to the indices page http://localhost:5601/app/enterprise_search/content/search_indices
+* Click on `Create a new index`, choose `Use the API`, and follow the steps there to create a new Elasticsearch index with default settings
+* Go to the overview page for your newly created index, follow the steps there to verify your Elasticsearch index.  
+  Notes:
+    * Generate an API key, and you will use the API key for authentication and authorization for this specific Elasticsearch index
+    * Use your `hostname` and `port` from the service credentials of the Elasticsearch instance to build `ES_URL`
+      ```bash 
+      export ES_URL=https://<hostname:port>
+      ```
+    * Append `--cacert <path-to-your-cert>` to the cURL for SSL connection or append `--insecure` to the cURL commands to ignore the certificate
+    * If you are able to run the `Build your first search query` command at the last step, your Elasticsearch index has been set up successfully!
+
+## Step 4: Set up Watson Assistant search extension using Elasticsearch index
+* Provision a Watson Assistant instance from the [IBM cloud catalog](https://cloud.ibm.com/catalog/services/watsonx-assistant)
+* Create a new Assistant in the new experience
+* Add a Search extension to your Assistant
+    * Find the Search extension on the Integrations page. Please refer to https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-add-custom-extension for more details
+    * Choose Draft environment and Elasticsearch (Es)
+    * Find the credentials from the ICD instance service credentials page
+        * `Elasticsearch index` is the name of your index from the previous step
+        * `Elasticsearch url` must start with the `https://` protocol
+        * For example:  
+          <img src="assets/connect_to_elasticsearch_index.png" width="565" height="409" />
+    * In the next step to `Configure result content`, use valid Elasticsearch document fields to config `name`, `body` and `url`, for example:  
+      <img src="assets/config_result_content.png" width="530" height="360" />  
+      Note: `author` and `name` are two fields from the documents in the index created from the previous step. They will be presented in the search result or used by Conversational Search.
+
+* Use the Search extension for `No action matches` scenario
```

Comment:
These instructions work, but they are not quite the recommended approach, which involves adding a new step and then removing the existing first step.  The recommended approach is documented at https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-search-add#search-no-action-matches .  Soon it will be replaced by new functionality that enables search integration _automatically_, but that might still be a few sprints away.  For now, I would drop this entire section and replace it with a link to https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-search-add#search-no-action-matches

FWIW, the main reason the recommended approach is better is that the approach in this document keeps the "I'm afraid I don't understand. Please rephrase your question." message and puts it before each search result, as you can see in the screenshot.